### DISCUSSION TITLE
Benefits didnt avail even within 5 days

### DISCUSSION THREAD

**Discussion by FreeWallah** [community/community#145360] [GitHub Education]
### Hi everyone,

My request application was approved on 18/11/2024 but the benefits didnt avail till now.
As per [github docs](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits) I have raised a discussion.
![20241123_120152](https://github.com/user-attachments/assets/86a67ede-ba24-40bb-ad32-0808d6d0831d)

**Comment by davevad93** [comment]
Hi @FreeWallah ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @FreeWallah was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
